lib/internal/util/inspect.js

```diff
@@ -2557,7 +2557,9 @@ function formatProperty(ctx, value, recurseTimes, key, type, desc,
         }
         ctx.indentationLvl -= 2;
       } catch (err) {
-        const message = `<Inspection threw (${inspect(err)})>`;
+        ctx.indentationLvl += 2;
+        const message = `<Inspection threw (${formatValue(ctx, err, recurseTimes)})>`;
+        ctx.indentationLvl -= 2;
```

Comment:
We probably don't want to modify the object here, we could pass a new one
```suggestion
        const message = `<Inspection threw (${formatValue({
          __proto__: ctx,
          indentationLvl: ctx.indentationLvl + 2,
        }, err, recurseTimes)})>`;
```